CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ endif()
 
 if (BUILD_DEPS)
     list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/crt/aws-crt-cpp/crt/aws-c-common/cmake")
+    list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/crt/aws-c-iot/cmake")
```

Comment:
@giordzaws Do you know if I need this line? I don't remember I added this line. Maybe it is a result of a rebase.

Reply:
I don't think that line is required.